hathor/event/model/node_state.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from enum import Enum, auto
+
+
+class NodeState(Enum):
+    LOAD = auto()
```

Comment:
Maybe avoid using `auto()` since the value might change if we add intermediate states in the future.

Reply:
Makes sense. Changed in https://github.com/HathorNetwork/hathor-core/pull/555/commits/0a4cb8cabcfac6f176d3b1556dc6997393c6259c

Reply:
Thanks!